python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -2645,3 +2647,31 @@ def get_latest_data_version_record(
             materialization = next(iter(materializations), None)
 
         return materialization or observation
+
+    @public
+    def get_latest_materialization_code_versions(
+        self, asset_keys: Iterable[AssetKey]
+    ) -> Mapping[AssetKey, Optional[str]]:
+        """Returns the code version used for the latest materialization of each of the provided
+        assets.
+
+        Args:
+            asset_keys (Iterable[AssetKey]): The asset keys to find latest materialization code
+                versions for.
+
+        Returns:
+            Mapping[AssetKey, Optional[str]]: A dictionary with a key for each of the provided asset
+                keys. The values will be None if the asset has no materializations. If an asset does
+                not have a code version explicitly assigned to its definitions but was
+                materialization, Dagster assigns a randomly-generated string as its code version.
```

Comment:
"was materialization" -> "was materialized'

Reply:
Dagster uses the run id as the code version if none is on the def, might as well say that

Reply:
Got it - made these changes